Superlista/Components/ListCard.swift

```diff
@@ -140,17 +140,29 @@ struct ListCard: View {
                     }
                 }
                 .alert(isPresented: $showAlertDelete){
-                    Alert(title: Text("Remover \(dataService.currentList!.title)"), message: Text("DeleteListAlertText"), primaryButton: .cancel(), secondaryButton: .destructive(Text("DeleteListAlertButton"), action:{
-                        dataService.removeList(dataService.currentList!)
-                        showAlertDelete = false
+                    let title = (dataService.currentList != nil) ? dataService.currentList?.title : ""
```

Comment:
Sempre que tiver uma "", se for um nome de lista coloca "NovaLista" (ou o localizable equivalente), e quando for um nome de usuário, chamar a função getNickname()